app/models/event.rb

```diff
@@ -0,0 +1,9 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  validates :title, :description, :date_time, :location, presence: true
```

Comment:
- [ ] [OPTIONAL] Make your project more professional by adding specific validations to title, description, and location, checking for minimum length, maximum length, format, etc.